bundles/AdminBundle/Resources/public/js/lib/ckeditor/config.js

```diff
@@ -7,4 +7,5 @@ CKEDITOR.editorConfig = function( config ) {
 	// Define changes to default configuration here. For example:
 	// config.language = 'fr';
 	// config.uiColor = '#AADC6E';
+    config.extraPlugins = 'wordcount,notification';
```

Comment:
Maybe It's anyway a good idea to just use the build config and then build CKEDITOR again (see https://github.com/pimcore/pimcore/blob/697e81cc40f4c879dd638d4656d7926bb80b8fa5/bundles/AdminBundle/Resources/public/js/lib/PIMCORE_MODIFICATIONS.md#L1) to include the additional plugins. 
Then this line shouldn't be necessary, because they are loaded in `ckeditor.js`.

Reply:
Hey @brusch, I've rebuilt CKEditor as you asked me to do. My flow was uploading the existing `build-config.js` file, then adding the wordcount plugin to the builder and downloading the optimized version which I then added to my branch instead of the existing. I've tested it locally beforehand and it worked fine. Pushed the changes to this branch.